modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -24,6 +24,9 @@
 import cloudpickle as pkl
 from dask.distributed import wait
 
+from distributed.client import _get_global_client
```

Comment:
@dchigarev , why did you import `_get_global_client`? I believe we can use `get_client` that is imported above.

Reply:
Agree with you. Also, there are more places where this function is used without the need. Created an issue about that (#3133)